translations/fr.json

```diff
@@ -0,0 +1,224 @@
+        "There was an error processing your request.": "Il y a eu une erreur dans le traitement de votre demande.",
+        "These values are used to determine craft/barter/price unlocks and flea market fees.": "Ces valeurs sont utilisées pour déterminer les déblocages d'artisanat/barter/prix et les frais de marché aux puces.",
+        "Time": "Temps",
+        "Too many notes to display.": "Trop de notes à afficher.",
+        "Total": "Total",
+        "Trader restocks": "Le marchand se réapprovisionne",
+        "Traders": "Marchands",
+        "Unknown": "Inconnu",
+        "Value": "Valeur",
+        "Velo": "IDK",
+        "Want to check the status of our services (api, website, bot, etc)?": "Vous voulez vérifier le statut de nos services (api, site web, bot, etc.) ?",
+        "Want to contribute to the bot or checkout the source code? View the project on GitHub!": "Vous voulez contribuer au Bot ou jeter un œil sur le code source ? Consultez le projet sur GitHub !",
+        "Weight": "Poids",
+        "Where [command] is one of the following commands:": "IDK",
+        "You can learn more about the bot's commands by entering:": "Vous pouvez en savoir plus sur les commandes du Bot en tapant:",
+        "You do not have any saved progress. Below are the defaults used to determine craft/barter/price unlocks and flea market fees.": "Vous n'avez pas de progression enregistrée. Vous trouverez ci-dessous les valeurs par défaut utilisées pour déterminer les déblocages d'artisanat/de troc/de prix et les frais de marché aux puces.",
+        "You have restock alerts enabled for": "Vous avez activé les alertes de réapprovisionnement pour",
+        "You must be an administrator to set channel restock alerts.": "Vous devez être un administrateur pour définir les alertes de réapprovisionnement des canaux.",
+        "You must supply your [TarkovTracker API token](https://tarkovtracker.io/settings/) to link your account.": "Vous devez fournir votre [jeton API TarkovTracker] (https://tarkovtracker.io/settings/) pour lier votre compte.",
+        "You need to specify a search term": "Vous devez spéficfier un terme de recherche",
+        "You need to specify an ammo type": "Vous devez spécifier un type de munitions",
+        "Your hideout progress will update from TarkovTracker {{updateTime}}.": "La progression de votre refuge sera mise à jour à partir de TarkovTracker {{updateTime}}."
+    },
+    "command": {
+        "about": "À propos",
+        "about_desc": "Vous donne quelques informations sur le Bot",
+        "alert": "alerte",
+        "all_desc": "Tout",
+        "ammo": "munitions",
+        "ammo_desc": "Obtenir des informations sur les munitions",
+        "ammo_name_desc": "Entrez le type de munitions",
+        "barter": "troquer",
+        "barter_desc": "IDK",
```

Comment:
"barter_desc" in this context means "barter description". Where we are describing information about a "barter" or "trade"

Reply:
You can look at the barter_desc value in the English translation for an idea of what it's meant to be: https://github.com/the-hideout/stash/blob/main/translations/en.json